packages/gatsby/src/commands/develop.js

```diff
@@ -49,10 +51,13 @@ const rlInterface = rl.createInterface({
 
 // Quit immediately on hearing ctrl-c
 rlInterface.on(`SIGINT`, () => {
-  telemetry.trackCli(`DEVELOP_STOP`)
   process.exit()
 })
 
+onExit(() => {
```

Comment:
which of these is run first, on SIGINT or on exit. I.e. will this hook exec from `SIGINT` + `process.exit` from above?

Reply:
@jamo SIGINT is run first. And `onExit` gets called with the call to `process.exit`. Tested this and moved the call to telemetry in `onExit` since that gets called gracefully even in case of the process getting killed by kill `<pid>`, for example.